contributors/design-proposals/api-machinery/aggregated-api-servers.md

```diff
@@ -221,8 +221,9 @@ mechanism.
 
 #### Limit ranges and resource quotas
 
-kubernetes server maintains [resource quotas](admission_control_resource_quota.md) and
-[limit ranges](admission_control_limit_range.md) for all resources.
+kubernetes server maintains [resource quotas](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/admission_control_resource_quota.md) and
+[limit ranges](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/admission_control_limit_range.md
```

Comment:
same for this: `/contributors/design-proposals/resource-management/admission_control_limit_range.md`